apps/aecore/src/aec_tx_sign.erl

```diff
@@ -11,40 +12,45 @@
 -include("common.hrl").
 -include("txs.hrl").
 
-data(SignedTx) ->
-    SignedTx#signed_tx.data.
+data(#signed_tx{data = Data}) ->
+    Data.
+
+signatures(#signed_tx{signatures = Sigs}) ->
+    Sigs.
 
 verify(failed_tx) ->
-    %% TODO: Verify signed txs
     {error, verification_failed};
-verify(_Tx) ->
-    ok.
+verify(#signed_tx{data = Data, signatures = Sigs}) ->
```

Comment:
I see `aec_tx_sign:verify` is called:
* When inserting tx in mempool;
* When building block candidate.

I believe it shall be called also when receiving block - but that is meant to be separate ticket I understand. We shall ensure this is tracked as ticket.

----

I understand all current transactions (coinbase and spend) need to be signed only by one party. When a tx requiring signatures from more than a party (if any such tx) will be implemented, this may require change (e.g. block shall contain only tx with 2 signature, while tx in mempool may have signature of only the originating party for enabling broadcast).
* Considering tx signed with only one party in this PR is ok - no changes needed;
* I suggest that a new ticket (or PR following up this) tracks hardcoding the expectation that tx has exactly one signature, so that it is clear that only txs with one signature is supported.